src/classes/logstream.class.php

```diff
@@ -1322,4 +1381,4 @@ private function ConvertMessageTypeString($szValue)
 	}
 
 }
-?>
\ No newline at end of file
+?>

```

Comment:
only cosmetic, could you remove the extra new line at the end? This can cause problems when headers have not been send yet.